lib/virt_feature_test_base.pm

```diff
@@ -196,4 +196,13 @@ sub get_active_pool_and_available_space {
     return ($active_pool, $available_size);
 }
 
+sub get_active_disk_and_available_space {
+    # ensure the available disk space size for active disk - /var/lib/libvirt
+    my $active_disk    = script_output 'lsblk -rnoPKNAME $(findmnt -nrvoSOURCE /var/lib/libvirt)';
+    my $available_size = script_output("df -k | grep libvirt | awk '{print \$4}'");
+    # default available disk unit as GiB
+    $available_size = int($available_size / 1048576);
+    return ($active_disk, $available_size);
+}
```

Comment:
Any room to make this function more proper for general purpose use and change name to, for example, get_folder_free_space ?  If you do not want to and think it is not necessary then could you use more specific function name instead of this general one "get_active_disk_and_available_space" ?
Anyway making some changes is not difficult

sub get_active_disk_and_available_space {
    my interested_folder = @_;
   die if ! defined interested_folder or interested == '';
    my $active_disk    = script_output 'lsblk -rnoPKNAME $(findmnt -nrvoSOURCE interested_folder)';
}

Reply:
Good point. We have the same idea here. 
For now,  this PR (or this `get_active_disk_and_available_space` fuc) worked for https://progress.opensuse.org/issues/78986
ensure the libvirt dir - /var/lib/libvirt before virt_autotest. 
This fuc had the following ways: 
1. Ensure the disk name(e.g sda) which was deployed /var/lib/libvirt disk partition on SLE15+ x86_64
2. Ensure the available| free disk space for deployed /var/lib/libvirt disk partition
So, if there was not any new request would keep the current code, imho.

Reply:
Could you help change function name to something else ?  get_libvirt_mounted_disk_and_freesapce ? It has nothing to do with "active"

Reply:
sure, I will change function name here, as your mentioned.

Reply:
sure, had updated name as `get_virt_disk_and_available_space()`